exercises/5.0/src/.rosinstall

```diff
@@ -1,6 +1,4 @@
-- git: {local-name: universal_robot, uri: 'https://github.com/ros-industrial/universal_robot.git', version: kinetic-devel}
+﻿- git: {local-name: universal_robot, uri: 'https://github.com/ros-industrial/universal_robot.git', version: kinetic-devel}
 - git: {local-name: fake_ar_publisher, uri: 'https://github.com/jmeyer1292/fake_ar_publisher.git', version: master}
-- git: {local-name: descartes, uri: 'https://github.com/ros-industrial-consortium/descartes.git', version: kinetic-devel}
-- git: {local-name: ur_modern_driver, uri: 'https://github.com/geoffreychiou/ur_modern_driver.git', version: iron-kinetic-devel}
-
-
+- git: {local-name: descartes, uri: 'https://github.com/ros-industrial-consortium/descartes.git', version: melodic-devel}
+- git: {local-name: ur_modern_driver, uri: 'https://github.com/ros-industrial/ur_modern_driver.git', version: kinetic-devel}
```

Comment:
Has this driver been tested with our demos on the real robot?

Reply:
Yes, we tested yesterday. That's where the changes in #250 came from